terraform/modules/bucket/main.tf

```diff
@@ -99,19 +100,24 @@ resource "aws_s3_bucket_server_side_encryption_configuration" "this" {
   }
 }
 
-data "aws_iam_account_alias" "current" {}
+data "aws_ssm_parameters_by_path" "cdap" {
+  path = "/cdap"
+  recursive = true
+}
```

Comment:
Using the `aws_ssm_parameters_by_path` data source allows for empty results. Paired with the call to the `zipmap()` in `local.cdap_ssm` **and** the `lookup()` on the same for `local.access_logs_bucket`, this helps us avoid failures on missing configuration and obviates the need for hard-coding bucket-access-logs buckets.